monitor/instance/docker/monitor.go

```diff
@@ -565,8 +565,21 @@ func (d *dockerMonitor) startDockerContainer(dockerInfo *types.ContainerJSON) er
 	if err := d.config.PUHandler.CreatePURuntime(contextID, runtimeInfo); err != nil {
 		return err
 	}
-
-	if err := d.config.PUHandler.HandlePUEvent(contextID, tevents.EventStart); err != nil {
+	var event tevents.Event
+	if dockerInfo.State.Paused {
+		event = tevents.EventPause
+	} else if dockerInfo.State.Running {
+		event = tevents.EventStart
+	} else if dockerInfo.State.Dead {
+		event = tevents.EventStop
+	} else {
```

Comment:
This else is a catch all - are there other event types that could come, if yes they should be dealt with errors.